sphinx_gallery/gen_gallery.py

```diff
@@ -1691,7 +1692,6 @@ def setup(app):
     app.connect("config-inited", post_configure_jupyterlite_sphinx, priority=900)
 
     if "sphinx.ext.autodoc" in app.extensions:
-        app.connect("autodoc-process-docstring", touch_empty_backreferences)
```

Comment:
Not sure about this @larsoner 
Now that we don't use the '.examples' files this is not required for us. But is it possible some 3rd party relies on these empty '.examples' files?!

Reply:
> But is it possible some 3rd party relies on these empty '.examples' files (created for objects that are never used in any example)?!

Yeah it probably is, otherwise Sphinx will complain about `.. include::` files that are missing

Reply:
Alright i will add this back in!